AdyenCard/Components/Card/CardViewControllerItemsProvider.swift

```diff
@@ -30,7 +30,7 @@ extension CardViewController {
         private let addressViewModelBuilder: AddressViewModelBuilder
 
         internal init(formStyle: FormComponentStyle,
-                      payment: Payment?,
+                      payment: Payment? = nil,
```

Comment:
let's not use default nil values, because we don't want to miss passing it by mistake :- )

Reply:
In this case it is internal, but sure - good point!